config.yml.erb

```diff
@@ -417,7 +417,7 @@ levelbuilder_mode:
 # make it easier to tell the difference
 use_local_header_color:
 
-default_hoc_mode: soon-hoc  # overridden by 'hoc_mode' DCDO param, except in :test
+default_hoc_mode: actual-hoc  # overridden by 'hoc_mode' DCDO param, except in :test
```

Comment:
is the DCDO flag set on production? I guess I'm worried that the DCDO flag will override this?

Reply:
Just spoke to Dani- Apparently we don't even need to set actual-hoc in Test because we aren't running tests or deploying. Closing this PR- thank you for the review and I'm sorry I didn't understand this sooner!